drivers/android-driver/build.gradle

```diff
@@ -6,11 +6,6 @@ archivesBaseName = 'sqldelight-android-driver'
 android {
   compileSdkVersion versions.compileSdk
 
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_7
```

Comment:
Seems out of scope for this change.

Reply:
Ahhh, I see it. It's the gh-pages branch. For OkHttp we put our mkdocs stuff master and use a script to put things into place.

Reply:
ooooooo thats very nice

can we do this too @saket

Reply:
@swankjesse sorry I was trying to change the source branch of this PR to be rebased on top of `gh-pages` and that took me a bit to figure out. The millions of irrelevant code changes should be gone now. 

`deploy_website.sh` looks very handy. @AlecStrong let's do it. I'll come pair with you in office?

Reply:
@swankjesse sorry I wasn't sure which base branch to target so I left it as is yesterday to figure out later. I changed it back to master so that people arent surprised. The millions of irrelevant code changes should be gone now. 

`deploy_website.sh` looks very handy. @AlecStrong let's do it. I'll come pair with you in office?

Reply:
We should fix the gh-pages branch. It shouldn't share a history with `master` and be totally devoid of code. We can do it once this lands and the script is ready.

Reply:
Added `deploy_website.sh`. If anyone's interested, I deployed the new docs to my fork: https://saket.github.io/sqldelight/